Inspicio/Controllers/ImagesController.cs

```diff
@@ -17,56 +17,45 @@ namespace Inspicio.Controllers
     [Authorize]
     public class ImagesController : Controller
     {
-        private readonly UserManager<ApplicationUser> _userManager;
+        private readonly UserManager<ApplicationUser> UserManager;   
```

Comment:
TitleCase for private fields is a bit unusual the previous naming style of `_userManager` (or even just `userManager` without the `_`) is more common in the C# world. TitleCase for public fields (and all properties) is common though.

You can see examples from [`List<T>` in the .NET framework](http://referencesource.microsoft.com/#mscorlib/system/collections/generic/list.cs,40) and from [AbstractAsynchronousTaggerProvider in the .NET Compiler](https://github.com/dotnet/roslyn/blob/614299ff83da9959fa07131c6d0ffbc58873b6ae/src/Compilers/Core/Portable/CodeGen/Win32Res.cs#L12) (there's lots more if you look around on GitHub)